web/src/stories/tags-input/base.stories.tsx

```diff
@@ -9,7 +9,7 @@ export default {
 }
 
 export const Base: React.FC<TabsProps> = props => (
-  <TagsInput {...props}>
+  <TagsInput onRemove={() => alert('Remove')} {...props}>
```

Comment:
I think the TagsInout should actually provide this out of the box:
there should be no need to define `onRemove` here, but pressing backspace should automatically remove the last `Tag`

Reply:
Agree